### DISCUSSION TITLE
Real time price update

### DISCUSSION THREAD

**Discussion by Synthacon** [osmosis-labs/osmosis-frontend#76] [Q&A]
HI,
Just wanted to ask that is there any way to get current price for ATOM/UST UST/ATOM (or any other pair) without updating the whole page?  Also it seem generally that to be able to see fresh price one has to refresh the whole page and chose the coin pare again and again. Would be nice to get list of my favorite coin pairs or why not just have constantly updating price? 
Cheers!

**Comment by CMarshall92** [comment]
As much as this would be awesome that's quite an architectural shift that would likely require a whole bunch of recatoring on the backend and websocket integration on the front end. Sure it can be done in time.
